### DISCUSSION TITLE
#24. VATZ biweekly meeting at 2022-12-21 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#331] [General]
### 1. Overall
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/6308023/206331691-542c8d53-32d9-473e-93ac-0438ffc95edc.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
22 | 2/17 | 11
23 | 1/15 | 6
---

# Poll
1. Vatz Dockerizing

**Comment by xellos00** [comment]
### Which one prefer for Dockerizing Vatz
1. Single docker per a repository : ❤️  
2. Individual Docker per plugins in single repository : 🚀 

---

### Option.1
1. Image per plugin
2. Docker run option isn't easy
   - Option documents
   - What to write on docker images  

### Option.2
1. Cons: Too many docker image 
2. Pros:  Docker run is easy

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/312
- https://github.com/dsrvlabs/vatz/issues/314
- https://github.com/dsrvlabs/vatz/issues/338
- https://github.com/dsrvlabs/vatz/issues/319
   - https://github.com/dsrvlabs/vatz/issues/318

---

- https://github.com/dsrvlabs/vatz/issues/341

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/vatz/issues/236
    - https://github.com/dsrvlabs/vatz/issues/326  
- Alarm of balance

**Reply by xellos00** [reply]
- Only Test Code left
- Monitoring balance(only injective)

**Comment by rootwarp** [comment]
# Joonkyo

- https://github.com/dsrvlabs/vatz/issues/316
  - https://github.com/dsrvlabs/vatz/issues/320

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/339
- https://github.com/dsrvlabs/vatz/issues/273
- https://github.com/dsrvlabs/vatz/issues/271
   -  https://github.com/dsrvlabs/vatz-plugin-eth-watcher/pull/1

**Reply by xellos00** [reply]
@dsrvlabs/validator 
Please, Share the idea for an issue 
- #333

**Comment by gnongs** [comment]
# 진홍
- #272 
- https://github.com/dsrvlabs/vatz/issues/342
  - #343 
- #329

**Reply by xellos00** [reply]
@xellos00 
- create a new issue for validators to upload: what to monitoring

**Comment by heejin-github** [comment]
## 희진
- #304 
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/23
- #327 
  - https://github.com/dsrvlabs/vatz-plugin-sysutil/pull/7

**Comment by meetrick** [comment]
# 황제
- no issue

**Comment by xellos00** [comment]
@qkrybin 
request documentation issue after 12/28/2022

**Reply by qkrybin** [reply]
Are these all of the documents that need to be updated? let me know if I missed something.
- Installation.md : add guide for plugin cmd
- contribution.md : the external process section is empty
- workflow.md : empty (don't know what it is for yet)
- Readme.md(main page)
